### DISCUSSION TITLE
Using FindTransferAnchors on individual snATAC + snRNA dataset pairs before merging?

### DISCUSSION THREAD

**Discussion by mattbcvs** [stuart-lab/signac#1718] [General]
I have a dataset of multiple tissue samples with paired snRNA and snATAC coming from each individual tissue. The format is 10x without any Multiome approach. I would like to link peaks to a panel of cell-specific genes using LinkPeaks.

I'm wondering whether FindTransferAnchors would work best if done per snRNA/snATAC pair rather than combining all samples together? I can therefore ensure anchors are formed for cells derived from the same tissue rather than across different tissues.

I would then have several co-embedded snRNA/ATAC objects, one per tissue, which I could then merge and run LinkPeaks(). Presumably I would have to unify the peak annotations across tissues at some point too.

I am not seeing any examples of this anywhere so looking for some feedback on this approach before I get started (it's a huge dataset)

**Comment by timoast** [comment]
Hi @mattbcvs, you could do as you suggest: integrate the scATAC-seq and scRNA-seq for each tissue, transfer gene expression values from the scRNA-seq to the scATAC-seq cells then run `LinkPeaks()`. Bear in mind that this will be using imputed data to find correlations between gene expression and peak accessibility, and the accuracy of that imputation would be based on the correlation of DNA accessibility in gene bodies with actual expression of the gene, and how accurately cells are matched across the RNA/ATAC experiments.

If you do decide to go with this type of analysis, you should certainly be cautious in interpreting the peak-gene linkage results, and make sure to independently validate any important results!
